src/hooks/useSearchSubmit.ts

```diff
@@ -27,18 +39,7 @@ export function useSearchSubmit(): (query: string) => void {
       const pathPrefix = prefixNetwork ? `${prefixNetwork}/${path}` : `${path}`
 
       if (path === 'address' && isEns(query)) {
-        if (web3) {
-          web3.eth.ens
-            .getAddress(query)
-            .then((res) => {
-              if (res && res.length > 0) {
-                history.push(`/${pathPrefix}/${res}`)
-              } else {
-                history.push(`/${pathPrefix}/${query}`)
-              }
-            })
-            .catch(() => history.push(`/${pathPrefix}/${query}`))
-        }
+        history.push(`/${path}/${query}`)
```

Comment:
I have prioritized the search for **mainnet** through the search engine, when it is an ENS it will search in Mainnet, I would like to know your opinion about this @alfetopito.

Or would it be better to keep the network so that I can also comply with #895

Reply:
I think this is fine for now.

If this becomes a necessity we can do the query per network (if available) since it's not possible to guarantee a mainnet address exists on rinkeby/xdai networks.